### DISCUSSION TITLE
Multiple NodeViewContent

### DISCUSSION THREAD

**Discussion by kenanchristian** [ueberdosis/tiptap#2106] [General Questions] [answered]
Hi there, I'm wondering is it possible to have a multiple `NodeViewContent` under the same `NodeViewWrapper`?

I'm looking into creating a 2 column layout with editable content inside each column.
Tried adding 2 content holes in `renderHTML` but this error `RangeError: Multiple content holes`

**Comment by philippkuehn** [comment, marked as answer]
No, this is not supported by ProseMirror. You have to define multiple nodes for that.

**Reply by kenanchristian** [reply]
Alright, got it. Thanks for the clarification!

**Reply by hivokas** [reply]
@kenanchristian how have you ended up implementing 2 column layout?

**Reply by malakhovmisha** [reply]
@kenanchristian Can you suggest a simple implementation example?

**Reply by kjfeng** [reply]
Would also like to know if anyone's figured out a good approach!

**Reply by nperez0111** [reply]
To match my reply in Discord:

Figure is a good example of what you are looking for that is simpler than the table extension: https://github.com/ueberdosis/tiptap/blob/f387ad3dd4c2b30eaea33fb0ba0b42e0cd39263b/demos/src/Experiments/GenericFigure/Vue/figure.ts#L4

You have a parent element that enforces that it must have certain content as it's children, so you are guaranteed in this case that there will be a single block element and then a figcaption.

**Reply by FengBuPi** [reply]
do you have a simple implementation example?plz help

**Reply by FengBuPi** [reply]
I have a solution, look this [https://github.com/ueberdosis/tiptap/discussions/7286](https://github.com/ueberdosis/tiptap/discussions/7286)
